### PR TITLE
Update config.schema.json

### PR DESCRIPTION
**Pull request by donavanbecker**
- Add Name config so that name is used in the logs instead of the full plugin name.
- removed default for type on deviceoverridee as it was inputting "Boolean" every time you save config

### CODE REVIEW

File: config.schema.json [hunk 1]
```diff
@@ -5,6 +5,11 @@
 	"headerDisplay": "",
 	"footerDisplay": "",
 	"schema": {
+		"name": {
+        "type": "string",
```
Comment (0x5e) [inline config.schema.json]: the file is using tab and L9-L12 you committed is space

**Review by 0x5e — CHANGES_REQUESTED** [review]

**Comment by 0x5e** [comment]
thanks @donavanbecker , there's some issue with tab & space, others are okay :)

**Comment by 0x5e** [comment]
I didn't know I can update your branch... is okay now.... thanks again!

**Review by 0x5e — APPROVED** [review]

**Comment by donavanbecker** [comment]
I will be posting some more changes soon :)

**Comment by donavanbecker** [comment]
See #285 and #283

**Pull request merged**